test/autodiff.jl

```diff
@@ -60,7 +60,25 @@
         gradient!(td, rand(2))
         hessian!(td, rand(2))
     end
-    @testset "autodiff ℝ → ℝᴺ" begin
+    for autodiff in (:finite, :forward)
```

Comment:
Should the outputs here test got the values, or is it just to make sure nothing errors?

Reply:
Minus "got", oops.

Yeah I was just wondering if it would be smart to test that the outputs (value, gradient/Jacobian etc) have the values we expect.

Reply:
It is very smart :)